ras/sosreport.py

```diff
@@ -38,6 +38,13 @@ def run_cmd_out(cmd):
         return process.system_output(cmd, shell=True, ignore_status=True,
                                      sudo=True).decode("utf-8")
 
+    def run_cmd_search(self, cmd, search_str):
+        self.output = self.run_cmd_out(cmd)
+        for line in self.output.splitlines():
+            if search_str in line:
+                search_line = line.split()
```

Comment:
can you add a break after this line so that once the string is found the loop can exit immediately?

Reply:
actually I want  to assign the last match found.

Reply:
if that is the case you can probably add `reversed(self.output.splitlines())` and add a break, that would reduce iterating through the lines significantly.

Reply:
sre

Reply:
sure